npiperelay.go

```diff
@@ -145,7 +259,7 @@ func main() {
 	}()
 
 	_, err = io.Copy(os.Stdout, conn)
-	if errors.Is(err, windows.ERROR_BROKEN_PIPE) || errors.Is(err, windows.ERROR_PIPE_NOT_CONNECTED) {
+	if underlyingError(err) == windows.ERROR_BROKEN_PIPE || underlyingError(err) == windows.ERROR_PIPE_NOT_CONNECTED {
```

Comment:
As mentioned, the function is not needed, so this line does not need to be changed.